src/core/core.scale.js

```diff
@@ -1206,9 +1206,19 @@ export default class Scale extends Element {
       const color = optsAtIndex.color;
       const strokeColor = optsAtIndex.textStrokeColor;
       const strokeWidth = optsAtIndex.textStrokeWidth;
+      let tickTextAlign = textAlign;
 
       if (isHorizontal) {
         x = pixel;
+
+        if (textAlign === 'left-right') {
+          if (i === ilen - 1) {
+            tickTextAlign = 'right';
+          } else {
+            tickTextAlign = 'left';
+          }
```

Comment:
Might be a better idea to change this to an else if block, check if it is the last tick then change the alignment. This way you can change the name of alignment to `inner` for example so all ticks align within the chart area. 

This way if you want you outer ticks to be inside but the others (if you have them) nicely centered that also still works. Makes it a bit more flexible

Reply:
Thanks for suggestions! Very good idea!